exercises/binary-search/.meta/gen.go

```diff
@@ -46,12 +70,14 @@ var testCases = []struct {
 	slice		[]int
 	key		int
 	x	int
+	err	string
 }{ {{range .J.Cases}}
 {
 	description:	{{printf "%q"  .Description}},
 	slice:		{{printf "%#v" .Input.Array}},
 	key:		{{printf "%d"  .Input.Value}},
-	x:	{{printf "%d"  .Expected}},
+	x:	{{printf "%d"  .Expected.ValueInt}},
+	err:	{{printf "%q"  .Expected.ValueString}},
```

Comment:
Perhaps align the {{printf ...   using tabs.  The template text string isn't subject to 'go fmt', but it looks better aligned (to me at least).  Maybe also the types for x, and err within the struct above.